supervisord.conf

```diff
@@ -0,0 +1,42 @@
+; Sample supervisor config file.
+;
+; For more information on the config file, please see:
+; http://supervisord.org/configuration.html
+;
+; Notes:
+;  - Shell expansion ("~" or "$HOME") is not supported.  Environment
+;    variables can be expanded using this syntax: "%(ENV_HOME)s".
+;  - Comments must have a leading space: "a=b ;comment" not "a=b;comment".
+
+[unix_http_server]
+file=/tmp/supervisor.sock  ; (the path to the socket file)
+
+[inet_http_server]         ; inet (TCP) server disabled by default
+port=127.0.0.1:9001        ; (ip_address:port specifier, *:port for all iface)
+username=admin             ; (default is no username (open server))
+password=admin             ; (default is no password (open server))
+
+[supervisord]
+logfile=/tmp/supervisord.log ; (main log file;default $CWD/supervisord.log)
+logfile_maxbytes=50MB        ; (max main logfile bytes b4 rotation;default 50MB)
+logfile_backups=10           ; (num of main logfile rotation backups;default 10)
+loglevel=info                ; (log level;default info; others: debug,warn,trace)
+pidfile=/tmp/supervisord.pid ; (supervisord pidfile;default supervisord.pid)
+nodaemon=true                ; (start in foreground if true;default false)
+minfds=1024                  ; (min. avail startup file descriptors;default 1024)
+minprocs=200                 ; (min. avail process descriptors;default 200)
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[supervisorctl]
+serverurl=unix:///tmp/supervisor.sock ; use a unix:// URL  for a unix socket
+
+[program:oppia-ml]
+command=python main.py                        ; the program (relative uses PATH, can take args)
+process_name=%(program_name)s_%(process_num)s ; process_name expr (default %(program_name)s)
+numprocs=5                                    ; number of processes copies to start (def 1)
+stdout_logfile=out.log                        ; stdout log path, NONE for none; default AUTO
```

Comment:
@anmolshkl no there aren't separate log files for multiple instances of same process. All logs are dumped to same file. Supervisor fist buffers log of processes for some time and flushes all process log buffers to same file.